source/optional_features/delayed_capture/index.html.md.erb

```diff
@@ -31,18 +31,16 @@ There are 4 possible responses:
 
 <div style="height:1px;font-size:1px;">&nbsp;</div>
 
-| Response Code  | Meaning                    |
-| -------------- | -------------------------- |
-|204             | Capture request accepted   |
-|404             | `paymentId` does not exist |
-|409             | Capture request rejected   |
-|500             | Downstream error           |
+|Response code | Meaning |
+|--|--|
+|204| Your capture request was accepted. |
+|404| No payment matched the `paymentId` you provided. |
+|409| The payment cannot be captured because its `status` is not `capturable`. |
+|500| Something is wrong with GOV.UK Pay. Please [contact us](/support_contact_and_more_information/#contact-us). |
 
 <div style="height:1px;font-size:1px;">&nbsp;</div>
 
-Your service must send the delayed capture request within 120 hours (5 days) of the
-payment creation, regardless of how long your user takes to complete the
-payment flow. Otherwise, it will expire.
+Your service must send the delayed capture request within 120 hours (5 days) of creating the payment, regardless of how long your user takes to complete the payment flow. Otherwise the payment will expire. An expired payment has a `status` of `failed` and returns a [P0020 API error](/api_reference/#api-error-codes).
```

Comment:
I think this might be a bit confusing. The P0020 API error will be in the response if a service queries the payment (along with the `status` of `failed`). But putting here could be interpreted as saying that if a service tries to capture the payment, the response will include P0020.

Reply:
Thanks Alex - in https://github.com/alphagov/pay-tech-docs/pull/363/commits/2087b8e7721e53e2ed73eb64ff7058fed89cebc4 I've linked to the Payment flow section about expired payments instead.